### ISSUE TITLE
the level name is too big

### ISSUE THREAD

**Issue by Helloausername** [TpdeaX/BetterPause-Geode#18]
![Screenshot_20240222-123612_Geode](https://github.com/TpdeaX/BetterPause-Geode/assets/158708969/7fda8b95-2bd6-499e-94cd-a90b3a9dca81)

**Comment by TpdeaX** [comment]
I don't see any problem I think xd

**Comment by Hanifinio** [comment]
yeah I think he just wants an option to make it smaller because it could look neater
